addons/account/models/res_config_settings.py

```diff
@@ -38,6 +38,24 @@ class ResConfigSettings(models.TransientModel):
     purchase_tax_id = fields.Many2one('account.tax', string="Default Purchase Tax", related='company_id.account_purchase_tax_id', readonly=False)
     tax_calculation_rounding_method = fields.Selection(
         related='company_id.tax_calculation_rounding_method', string='Tax calculation rounding method', readonly=False)
+    account_journal_suspense_account_id = fields.Many2one('account.account', string='Suspense Account',
+                                                          related='company_id.account_journal_suspense_account_id', readonly=False,
+                                                          domain=lambda self: "[('deprecated', '=', False), ('company_id', '=', company_id), \
+                                                          ('user_type_id.type', 'not in', ('receivable', 'payable')), \
+                                                          ('user_type_id', '=', %s)]" % self.env.ref('account.data_account_type_current_liabilities').id,
+                                                          help='Account used as automatic counterpart to bank/cash transactions')
+    account_journal_payment_debit_account_id = fields.Many2one('account.account', string='Outstanding Receipts Account', readonly=False,
+                                                               related='company_id.account_journal_payment_debit_account_id',
+                                                               domain=lambda self: "[('deprecated', '=', False), ('company_id', '=', company_id), \
+                                                               ('user_type_id.type', 'not in', ('receivable', 'payable')), \
+                                                               ('user_type_id', '=', %s)]" % self.env.ref('account.data_account_type_current_assets').id,
+                                                               help='Account used as automatic counterpart account to payments received')
+    account_journal_payment_credit_account_id = fields.Many2one('account.account', string='Outstanding Payments Account', readonly=False,
+                                                               related='company_id.account_journal_payment_credit_account_id',
+                                                               domain=lambda self: "[('deprecated', '=', False), ('company_id', '=', company_id), \
+                                                               ('user_type_id.type', 'not in', ('receivable', 'payable')), \
+                                                               ('user_type_id', '=', %s)]" % self.env.ref('account.data_account_type_current_assets').id,
+                                                               help='Account used as automatic counterpart account to payments sent')
```

Comment:
Do you really like the big whitespace on the left?
```
    account_journal_payment_credit_account_id = fields.Many2one(
        comodel_name='account.account',
        string="Outstanding Payments Account",
        readonly=False,
        related='company_id.account_journal_payment_credit_account_id',
        domain=lambda self: "[('deprecated', '=', False), ('company_id', '=', company_id), ('user_type_id.type', 'not in', ('receivable', 'payable')), ('user_type_id', '=', %s)]" % self.env.ref('account.data_account_type_current_assets').id,
        help='Account used as automatic counterpart account to payments sent')
``` 
is far more readable to me but this is a purely cosmetic. You can ignore this comment is you don't agree